src/main/java/de/tum/in/www1/artemis/domain/exam/StudentExam.java

```diff
@@ -0,0 +1,154 @@
+package de.tum.in.www1.artemis.domain.exam;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import de.tum.in.www1.artemis.domain.AbstractAuditingEntity;
+import de.tum.in.www1.artemis.domain.Exercise;
+import de.tum.in.www1.artemis.domain.User;
+
+@Entity
+@Table(name = "student_exam")
+public class StudentExam extends AbstractAuditingEntity {
```

Comment:
why do we have AbstractAuditingEntity here? I don't think this is necessary

Reply:
We thought that it might be interesting to know when the student's exam was created and when it was updated last time. You think we shouldn't track this here? 

I've removed `AbstractAuditingEntity` from `Exam` and `ExerciseGroup`.

Reply:
I imagine that they are only generated once. In case a 2nd Generation would be necessary, we would delete all existing objects and then recreate them based on the new config. Then we would have no use for auditing

Reply:
Okay, I will remove it.